src/FloraWeb/Server.hs

```diff
@@ -60,11 +63,24 @@ runServer floraEnv = withStdoutLogger $ \logger -> do
 floraServer :: Routes (AsServerT FloraM)
 floraServer = Routes
   { assets = serveDirectoryWebApp "./static"
-  , pages = \userInfo -> hoistServer (Proxy @Pages.Routes) (withReaderT $ \floraEnv -> CallInfo{..}) Pages.server
+  , pages = \session ->
+      hoistServer (Proxy @Pages.Routes)
+                  (withReaderT (const session) Pages.server)
   }
 
+{-
+  si j'ai un cookie, je le décode
+    -> récupère le session_id
+    -> récupère la session depuis la db
+  si j'ai pas de cookie, je créé une session 
+    -> créé un cookie vide de variables de templates et dépose-le 
```

Comment:
Interesting choice to document in french 😄

Reply:
That's become it's still a draft ;-)